src/services/safe_browsing.py

```
@@ -0,0 +1,394 @@
+        except Exception as e:
+            logger.exception("Safe Browsing API error")
+            if self._fail_open:
+                # fail-open: API障害時はURLを許可
+                for url in urls_to_check:
+                    results[url] = SafeBrowsingResult(
+                        url=url,
+                        is_safe=True,
+                        error=str(e),
+                    )
+            else:
+                # fail-close: API障害時はURLを拒否（例外を送出）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)